workers/data_refinery_workers/processors/array_express.py

```diff
@@ -144,19 +153,17 @@ def _create_result_objects(job_context: Dict) -> Dict:
     """ Create the ComputationalResult objects after a Scan run is complete """
 
     result = ComputationalResult()
-    result.command_executed = "'SCAN.UPC', 'SCANfast'" # Need a better way to represent this R code.
+    result.commands = ['SCAN.UPC', 'SCANfast'] # Need a better way to represent this R code.
```

Comment:
This should be `"SCAN.UPC::SCANfast"` because this is the same as `"SCAN.UPC::SCAN_TwoColor"`, just a different function from the SCAN.UPC package.